### PR TITLE
static-static should be serve-static (broken link)

### PR DESCRIPTION
**Pull request by aeharding**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ var app = express();
 app.use(kraken());
 app.listen(8000);
 ```
-
+s
```
Comment (aredridel) [inline README.md]: Whoops. Typo. Want to fix that up?

File: README.md [hunk 2]
```diff
@@ -176,7 +176,7 @@ Kraken comes with common middleware already included in its `config.json` file.
       - *String* - local path to the favicon file (default: `"path:./public/favicon.ico"`)
 * `"static"` - serves static files from a specific folder
   - Priority - 40
-  - Module - `"serve-static"` ([npm](https://www.npmjs.org/package/static-static))
+  - Module - `"serve-static"` ([npm](https://www.npmjs.org/package/serve-static))
```
Comment (aredridel) [inline README.md:179]: awesome, well spotted -- thank you!

**Comment by aeharding** [comment]
@aredridel sorry about the typo -- should be fixed now :+1:

**Comment by aredridel** [comment]
Thank you so much!

**Comment by aredridel** [comment]
:+1:

**Pull request merged**